[SlackAdapter][v1] slack helper command to activity

Konigstiger opened the pull request:
### Description
This PR has the implementation of the method command_to_activity_async in the class SlackHelper.

### Changes made
- Add method implementation.
- Adapt usings.
- Adapt docstring to reStructured Text.

### Testing
![image](https://user-images.githubusercontent.com/24591490/72835958-6eb86680-3c6a-11ea-83e3-5c9e3aca24dd.png)

Commit:
Update .gitignore

Commit:
SlackHelper: Add method command_to_activity

Changes:
- Add method command_to_activity.
- Changed docstring to reStructured Text.

Commit:
Updated values

Maihav commented on slack_adapter/slack_helper.py:
```diff
@@ -21,4 +24,35 @@ def activity_to_slack(activity: Activity) -> NewSlackMessage:
+        :return: An activity containing the event data.
+        """
+        if not slack_body:
+            raise Exception("slack_body")
```
I'm not sure if this is the proper way to manage exceptions when None is encountered as an argument. I found that most of the python function raises TypeError if None is passed as an argument. Can you check it?

Maihav commented on slack_adapter/slack_helper.py:
```diff
@@ -21,4 +24,35 @@ def activity_to_slack(activity: Activity) -> NewSlackMessage:
+                            text=slack_body.text,
+                            type=ActivityTypes.event)
+
+        activity.recipient.id = client.get_bot_user_by_team(activity, cancellation_token)
```
Can you check if we will need to implement an await here? As the C# version does. 

FYI: https://docs.python.org/3/library/asyncio-task.html#awaitables

Konigstiger replied:
You're right. The `await `keyword was needed; also, added `async` to the method declaration. Thanks!

Maihav commented:
I mentioned two topics to research, the rest LGTM

Commit:
Updated command_to_activity_async

Changes:
- Changed Exception for TypeError.

Commit:
Feedback applied on async / await

Changes:
- Added async to method declaration in command_to_activity_async.
- added await keyword in call to client.get_bot_user_by_team, that is a async method.
